src/com/sap/piper/PiperGoUtils.groovy

```diff
@@ -65,4 +68,40 @@ class PiperGoUtils implements Serializable {
         }
         return false
     }
+
+    /*
+     * The returned string can be used directly in the command line for retrieving the configuration via go
+     */
+    public String prepareConfigurations(List configs, String configCacheFolder) {
+
+        for(def customDefault : configs) {
+            steps.writeFile(file: "${additionalConfigFolder}/${customDefault}", text: steps.libraryResource(customDefault))
+        }
+        joinAndQuote(configs.reverse(), configCacheFolder)
+    }
+
+    /*
+     * prefix is supposed to be provided without trailing slash
+     */
+    private static String joinAndQuote(List l, String prefix = '') {
+
+        Iterable _l = []
```

Comment:
I am in favor of not using abbreviations.

Reply:
Now with more speaking names.